src/vm/page.c

```diff
@@ -273,11 +289,21 @@ void _page_showPages(void)
 				break;
 		}
 
-		if (rep >= 4)
+		if (rep >= 4) {
+			if ((w += page_digits(rep + 1, 10) + 3) > 80) {
+				lib_printf("\nvm: ");
+				w = page_digits(rep + 1, 10) + 7;
```

Comment:
It would be benefitial to cache `page_digits(rep + 1, 10)` result in a variable to invoke funtion only once.

Reply:
Fixed.